bus/client.go

```diff
@@ -158,7 +158,10 @@ func (c *Client) WalletPending() (resp []types.Transaction, err error) {
 // Hosts returns up to max hosts that have not been interacted with since
 // the specified time.
 func (c *Client) Hosts(notSince time.Time, max int) (hosts []hostdb.Host, err error) {
-	err = c.c.GET(fmt.Sprintf("/hosts?notSince=%v&max=%d", paramTime(notSince), max), &hosts)
+	values := url.Values{}
+	values.Set("notSince", paramTime(notSince).String())
+	values.Set("max", fmt.Sprint(max))
+	err = c.c.GET(fmt.Sprintf("/hosts?%s", values.Encode()), &hosts)
```

Comment:
I think the most expedient solution here is to ditch `url.Values` and make `paramTime.String()` return an escaped string.